app/controllers/v1/greetings_controller.rb

```diff
@@ -0,0 +1,10 @@
+module V1
+  class GreetingsController < ApplicationController
+    def index
+      random = Random.new
+      limit = Greeting.all.length
+      @greeting = Greeting.find(random.rand(1..limit)).description
+      render json: @greeting
+    end
```

Comment:
- [ ] When I run `http://localhost:3000/v1/greetings` in the browser it is showing the error kindly check what you are doing wrong
![err](https://user-images.githubusercontent.com/44745855/146277233-be2cecd3-6564-4658-a80a-a43f0332da6b.png)